app/src/main/java/org/mozilla/fenix/settings/logins/fragment/LoginDetailFragment.kt

```diff
@@ -77,7 +77,8 @@ class LoginDetailFragment : Fragment(R.layout.fragment_login_detail) {
             )
         }
         loginDetailView = LoginDetailView(
-            view.findViewById(R.id.loginDetailLayout)
+            view.findViewById(R.id.loginDetailLayout),
+            binding
```

Comment:
This seems strange, but again this `LoginDetailView` is not really a View but rather a delegate to handle binding.